scrapyrt/resources.py

```diff
@@ -74,9 +75,13 @@ def handle_error(self, exception_or_failure, request):
         return self.format_error_response(exception, request)
 
     def format_error_response(self, exception, request):
+        # Python exceptions don't have message attribute in Python 3+ anymore.
+        # Twisted HTTP Error objects still have 'message' attribute even in 3+
+        # and they fail on str(exception) call.
+        msg = exception.message if hasattr(exception, 'message') else str(exception)
```

Comment:
exception in Python 2 and 3 have `.args` attr, you can use `exc.args[0]`. can you do that with `Twisted HTTP Error objects` in both py2 and py3?